orchagent/pfcwdorch.cpp

```diff
@@ -560,10 +560,12 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
                 sai_serialize_object_id(queueId));
     }
 
+#if 0
```

Comment:
If you are certain, remove the code below and add a comment here.

Reply:
Removed, and added comment